lib/src/cnf/mod.rs

```diff
@@ -35,3 +35,9 @@ pub const SERVER_NAME: &str = "SurrealDB";
 
 /// Datastore processor batch size for scan operations
 pub const PROCESSOR_BATCH_SIZE: u32 = 50;
+
+/// Forward all signup/signin query errors to a client trying authenticate to a scope. Do not use in production.
+pub static INSECURE_FORWARD_SCOPE_ERRORS: Lazy<bool> = Lazy::new(|| {
+	let val = std::env::var("SURREAL_INSECURE_FORWARD_SCOPE_ERRORS").unwrap_or("false".to_string());
+	val.to_lowercase() != "false"
```

Comment:
Could we keep this consistent with the other functions that are going to be added in https://github.com/surrealdb/surrealdb/pull/3000:

```suggestion
 	let default = false;
 	std::env::var("SURREAL_INSECURE_FORWARD_SCOPE_ERRORS")
 		.map(|v| v.parse::<bool>().unwrap_or(default))
 		.unwrap_or(default)
```

Reply:
Yeah, I took inspiration from that, but didn't know that parse function worked that way. Even cleaner!